### ISSUE TITLE
[Enhancement] Display "current interval" on the 'Set Due Date' dialog

### ISSUE THREAD

**Issue by david-allison** [ankidroid/Anki-Android#16406]
Only when a single card is shown and the card is in the review state

https://forums.ankiweb.net/t/ankidroid-reschedule-card-pop-up-doesnt-show-current-interval-anymore/44809/

**Comment by SjorsO** [comment]
The "card info" dialog also shows current interval, but that displays in months. For example, if the interval is 48 days the "card info" dialog shows 1.6 months. It would be nice if the "set due date" dialog shows the interval in days, because values like 1.6 months are hard to grok.

**Comment by Giyutomioka-SS** [comment]
Can i work on this issue ?

**Comment by Giyutomioka-SS** [comment]
I've read Anki forums about this issue, but could you explain it to me once more? What exactly needs to be done here, and where should I start? @david-allison

**Comment by david-allison** [comment]
Look to the diff on:

* https://github.com/ankidroid/Anki-Android/pull/15699

See how 'current interval' was implemented, and re-implement it for new screen

**Comment by david-allison** [comment]
@Giyutomioka-SS are you still working on this?

**Comment by Giyutomioka-SS** [comment]
> @Giyutomioka-SS are you still working on this?

@david-allison Yeah, I had started re-implementing the current interval for the new screen but ran into a lot of errors. With uni exams coming up at that time, I had to step back for a bit. Could you give me a hint on the right approach to tackle this from scratch?

**Comment by david-allison** [comment]
Sorry, I don't have the capacity for this right now
